packages/pigeon/pigeons/core_tests.dart

```diff
@@ -87,6 +143,43 @@ abstract class HostIntegrationCoreApi {
   @ObjCSelector('echoUint8List:')
   Uint8List echoUint8List(Uint8List aUint8List);
 
+  // ========== Syncronous nullable method tests ==========
+
+  /// Returns the inner `aString` value from the wrapped object, to test
+  /// sending of nested objects.
+  @ObjCSelector('extractNestedNullableStringFrom:')
+  String? extractNestedNullableString(AllNullableTypesWrapper wrapper);
+
+  /// Returns the inner `aString` value from the wrapped object, to test
+  /// sending of nested objects.
+  @ObjCSelector('createNestedObjectWithNullableString:')
+  AllNullableTypesWrapper createNestedNullableString(String? nullableString);
+
+  /// Returns passed in arguments of multiple types.
+  @ObjCSelector('sendMultipleNullableTypesABool:anInt:aString:')
+  AllNullableTypes sendMultipleNullableTypes(
+      bool? aNullableBool, int? aNullableInt, String? aNullableString);
+
+  /// Returns passed in int.
+  @ObjCSelector('echoNullableInt:')
+  int? echoNullableInt(int? aNullableInt);
+
+  /// Returns passed in double.
+  @ObjCSelector('echoNullableDouble:')
+  double? echoNullableDouble(double? aNullableDouble);
+
+  /// Returns the passed in boolean.
+  @ObjCSelector('echoNullableBool:')
+  bool? echoNullableBool(bool? aNullableBool);
+
+  /// Returns the passed in string.
+  @ObjCSelector('echoNullableString:')
+  String? echoNullableString(String? aNullableString);
+
+  /// Returns the passed in Uint8List.
+  @ObjCSelector('echoNullableUint8List:')
+  Uint8List? echoNullableUint8List(Uint8List? aNullableUint8List);
+
```

Comment:
It doesn't have to be in this PR, but I would definitely include a `List` (vs the special list types like the above) and/or `Map` type here.

Reply:
Still lots to add here :) it's on it's way